src/nbody_basic_potentials.jl

```diff
@@ -0,0 +1,51 @@
+abstract type PotentialParameters
+end
+
+struct LennardJonesParameters <: PotentialParameters
+    ϵ::AbstractFloat
```

Comment:
All variables in a struct must have a concrete type. You can do this by parámetrising the type.

Reply:
You are right, it would be better for performance (I have just read the Performance tips again). 
Ok.